services/src/main/java/uk/gov/moj/sdt/services/messaging/QueueConfig.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.moj.sdt.services.messaging;
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+import java.util.Map;
+
+
+@Component
+@Data
+@EnableConfigurationProperties
+@ConfigurationProperties(prefix = "ms.servicebus.internal")
```

Comment:
Should there be an ms.servicebus.internal properties file included in the files?

Reply:
we don't need the file with this name, it looks for the properties in application.yml file with the prefix jms.servicebus.internal

Reply:
ms.servicebus.internal -> changed to jms.servicebus.internal

Reply:
Confirmed that prefix has been changed.